src/QMCDrivers/DMC/WalkerControl.cpp

```diff
@@ -475,7 +457,11 @@ void WalkerControl::swapWalkersSimple(MCPopulation& pop)
       {
         ScopedTimer local_timer(my_timers_[WC_recv]);
         myComm->comm.receive_n(awalker.DataSet.data(), byteSize, jobit->target);
-        awalker.copyFromBuffer();
+	auto walker_id = awalker.getWalkerID();
+	awalker.copyFromBuffer();
+	auto parent_id = awalker.getWalkerID();
+	awalker.setWalkerID(walker_id);
+	awalker.setParentID(parent_id);
```

Comment:
Could you explain all the setting?
`auto parent_id = awalker.getWalkerID();` why?
Are you sure the parent id also has been wired?

Reply:
Added test and discovered the answer was kinda.  It was for transfer but not for reading from WalkerConfigurations

Reply:
I have since discovered that it is also not written or read from walker hdf5 file. I'd rather address that in a future PR.